pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -140,10 +141,13 @@ func reconcileCronJob(ctx context.Context,
 	}
 
 	cronJob.Spec.JobTemplate.Spec.Template.Spec.Containers[0].Image = operatorImagePullSpec
+	// The "--etcd-backup-name=$MY_POD_NAME" arg sets the name of the EtcdBackup CustomResource to the name of the pod
+	// executing the EtcdBackup CR creation cmd. MY_POD_NAME is expected to be set via the downward API in
+	// the CronJob spec.
+	// This ensures we have a unique EtcdBackup CR name for each Job run of the CronJob's execution
 	cronJob.Spec.JobTemplate.Spec.Template.Spec.Containers[0].Args = []string{
-		"--backup-name=" + backup.Name,
-		"--retention-number=5",
-		"--pvc=" + backup.Spec.EtcdBackupSpec.PVCName,
+		"--etcd-backup-name=$MY_POD_NAME" + backup.Name,
```

Comment:
I have this as an explicit flag in case we change it but this could also just be read directly from the envvar in the pod like I'm doing for the ownerReference vars.